appserver/pom.xml

```diff
@@ -883,12 +883,12 @@
                             <artifact>
                                 <groupId>jakarta.servlet</groupId>
                                 <artifactId>jakarta.servlet-api</artifactId>
-                                <version>${servlet-api.version}</version>
+                                <version>${jakarta.servlet-api.version}</version>
                             </artifact>
                             <nonFinal>false</nonFinal>
                             <jarType>api</jarType>
-                            <specVersion>3.1</specVersion>
```

Comment:
However - the spec is only versioned to 2 components (like 6.0 in JakartaEE 10), while the `jakarta.servlet-api.version` property holds the API artifact version (with bugfix component). I see below for annotation it's kept in-sync with the same value/property, but I'm not sure what is the role of `<specVersion>` and if it should match the implementation to the point... I imagine it would not be here to keep duplicate...

But `3.1` is of course not good here as well :grin: